src/ExpressionEvaluator/Core/Source/ResultProvider/Helpers/InternalWellKnownMemberNames.cs

```diff
@@ -7,5 +7,6 @@ internal static class InternalWellKnownMemberNames
         public const string NullableHasValue = "hasValue";
         public const string NullableValue = "value";
         public const string SqlStringValue = "m_value";
+        public const string IntPtrAndUIntPtrStringValue = "m_value";
```

Comment:
This is _value in .NET Core

Reply:
if you're checking fields, you should probably check for both _value and m_value

Reply:
Thank you, @r-ramesh and @cston! Feedback addressed. Please take a look again.